src/ledger/accounts.ts

```diff
@@ -0,0 +1,71 @@
+// an accounting reminder:
+// https://en.wikipedia.org/wiki/Double-entry_bookkeeping
+
+// assets:
+export const assetsMainAccount = "Assets"
+export const bitcoindAccountingPath = `${assetsMainAccount}:Reserve:Bitcoind`
+export const lndAccountingPath = `${assetsMainAccount}:Reserve:Lightning` // TODO: rename to Assets:Lnd
+export const escrowAccountingPath = `${assetsMainAccount}:Reserve:Escrow` // TODO: rename to Assets:Lnd:Escrow
+
+// liabilities
+export const liabilitiesMainAccount = "Liabilities"
+export const accountPath = (uid) => `${liabilitiesMainAccount}:${uid}`
+export const resolveAccountId = (accountPath: string | string[]) => {
+  let id: string | null = null
+
+  if (!accountPath) {
+    return id
+  }
+
+  let path = accountPath
+
+  if (typeof accountPath === "string") {
+    path = accountPath.split(":")
+  }
+
+  if (
+    Array.isArray(path) &&
+    path.length === 2 &&
+    path[0] === liabilitiesMainAccount &&
+    path[1]
+  ) {
+    id = path[1]
+  }
+
+  return id
+}
+
+let cacheDealerPath: string
+let cachebankOwnerPath: string
+
+const throwError = (account) => Promise.reject(`Invalid ${account}AccountPath`)
+let bankOwnerResolver = (): Promise<string> => throwError("bankOwner")
+let dealerResolver = (): Promise<string> => throwError("dealer")
```

Comment:
struggling to understand what this is supposed to do, a promise that returns a Promise.reject?

Reply:
this is just to guarantee that there is an implementation outside the ledger like in https://github.com/GaloyMoney/galoy/blob/feat-ledger-refactor/src/mongodb.ts#L12 so if you did not pass the revolvers it will throw an exception